src/Option.ts

```diff
@@ -204,6 +225,27 @@ export class Some<A>
   toString() {
     return `some(${toString(this.value)})`
   }
+  contains(setoid: Setoid<A>, a: A): boolean {
```

Comment:
Would it make more sense to make `setoid` optional and fall back to `this.value.equals(a)`?